daprdocs/content/en/operations/components/pluggable-components/pluggable-components-overview.md

```diff
@@ -0,0 +1,122 @@
+
+<img src="/images/concepts-building-blocks.png" width=400>
+
+## Pluggable components vs. Built-in components
+
+Dapr provides two pathways for creating new components:
+
+- The pluggable component route
+- The built-in components route found in the [components-contrib repository ](https://github.com/dapr/components-contrib).
+
+While both component options leverage Dapr's building block APIs, each have different implementation processes.
+
+| Component details            | [Built-in Component](https://github.com/dapr/components-contrib/blob/master/docs/developing-component.md)  | Pluggable Components                                                                                                                                                                                                                                       |
+| ---------------------------- | :--------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Language**                 | Can only be written in Go                                                                                  | [Can be written in any gRPC-supported language](https://grpc.io/docs/what-is-grpc/introduction/#:~:text=Protocol%20buffer%20versions,-While%20protocol%20buffers&text=Proto3%20is%20currently%20available%20in,with%20more%20languages%20in%20development) |
+| **Where it runs**            | As part of the Dapr executable itself                                                                      | As distinct process, container or pod. Runs seperate from Dapr itself                                                                                                                                                                                      |
+| **Integration with Dapr**    | Integrated directly into Dapr codebase                                                                     | Integrates with Dapr via Unix Domain Sockets (using gRPC )                                                                                                                                                                                                 |
+| **Hosting**                  | Hosted in Dapr repository                                                                                  | Hosted in your own repository                                                                                                                                                                                                                              |
```

Comment:
What does hosting means here? I suggest deleting this, since these are not hosted into a Dapr repository.

Reply:
The idea here was to highlight the difference on code ownership and lifecycle management between built-in components and pluggable ones: . Perhaps "hosting" is the wrong term for it. How about we change this to:

```suggestion
| **Code ownership and management** | Hosted in Dapr repository, subjected to Dapr license, managed by Dapr team                                                                                  | Code hosted in your own repository, licensed under your terms, managed by your team |
```